backend/app/storage.py

```diff
@@ -25,17 +25,10 @@ async def get_assistant(user_id: str, assistant_id: str) -> Optional[Assistant]:
         )
 
 
-async def list_public_assistants(assistant_ids: Sequence[str]) -> List[Assistant]:
+async def list_public_assistants() -> List[Assistant]:
     """List all the public assistants."""
     async with get_pg_pool().acquire() as conn:
-        return await conn.fetch(
-            (
-                "SELECT * FROM assistant "
-                "WHERE assistant_id = ANY($1::uuid[]) "
-                "AND public = true;"
-            ),
-            assistant_ids,
-        )
+        return await conn.fetch(("SELECT * FROM assistant WHERE public = true;"))
```

Comment:
Double checking: I think the best practice (or technically correct) is to use the `IS` operator to evaluate boolean expressions.
```sql
WHERE public IS TRUE;
```
This might not matter.

Reply:
Sounds good. Changed.